src/schema/v2/ArtworkImport/__tests__/updateArtworkImportMutation.test.ts

```diff
@@ -0,0 +1,48 @@
+import gql from "lib/gql"
+import { runAuthenticatedQuery } from "schema/v2/test/utils"
+
+describe("UpdateArtworkImportMutation", () => {
+  it("updates an artwork import successfully", async () => {
+    const artworkImportUpdateLoader = jest.fn().mockResolvedValue({
+      artworkImportID: "artwork-import-1",
```

Comment:
This isn't right. The mock response of a data loader needs to be a mock of the gravity JSON payload from the API endpoint corresponding to the loader. Where's `id`, `location_id`, etc.?